src/frontend/src/components/DateField/index.tsx

```diff
@@ -14,19 +14,41 @@ interface Props {
   formik: FormikContextType<any>;
   helperText?: any;
   onChange?: (d: ChangeEvent) => void;
+  shouldShowEditedCellsAsMarked?: boolean;
 }
 
+export type valueType = string | boolean | NamedGisaidLocation | undefined;
+
 export default function DateField({
   fieldKey,
   formik,
   helperText,
   onChange,
+  shouldShowEditedCellsAsMarked,
```

Comment:
This prop name implies that the date field component knows about the concept of a "cell", which is shouldn't because it is an input field which could totally exist outside of a table. I'm wondering if there's a more generic name that you can think of for this prop?

Reply:
i'll change `shouldShowEditedInputAsMarked`